### PR TITLE
PADV-439 - Migrate the plugin following the migration guidelines

### PR DESCRIPTION
**Pull request by sergivalero20**
## Ticket

https://agile-jira.pearson.com/browse/PADV-439

## Description

This PR adds more description to the README file. 

## Type of Change

- [x] Add the current features 
- [x] Add how to use the plugin
- [x] Add how to test the plugin

## How to test

- Deploy a Devstack environment.
- Clone the plugin with the latest release.
- Install the plugin in the LMS and CMS containers.

## Reviewers

- [ ] @Squirrel18
- [ ] @anfbermudezme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +16,18 @@ Place your plugin features list.
 
 ## Usage
 
-Include a usage description for your plugin.
+To configure this plugin correctly you have to set this environment variables in your
+local machine like [here](https://github.com/Pearson-Advance/edx-deployment-pe/blob/juniper/ansible/environments/prod/group_vars/all/10_edxapp.yml#L47).
```
Comment (Squirrel18) [inline README.md]: This is private, this contains keys, you **should never** link this in public repositories... Just mention the settings that need to be configured. @sergivalero20

File: README.md [hunk 2]
```diff
@@ -15,7 +16,18 @@ Place your plugin features list.
...
+
+## Testing
+
+- In Studio add a new video component to a course.
+- Press the *Edit* button and then go to the *Advanced* section.
+- In the *Video File URLs* field add the protected video. (e,g,. /secure-cloudfront-video/?key=9780136721390/M101_01_08_08_01.mp4).
```
Comment (Squirrel18) [inline README.md]:
```suggestion
- In the *Video File URLs* field add the protected video. (e,g,. /secure-cloudfront-video/?key=9780136721390/M101_01_08_08_01.mp4).
```
This is a link to an actual protected resource, you **should never** link to a protected resource in a public repository. Just add an example of what the URL looks like. @sergivalero20

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sergivalero20) [reply README.md]: @Squirrel18  Yes, you're right!
I already removed it.

File: README.md [hunk 3]
```diff
@@ -15,7 +16,21 @@ Place your plugin features list.
...
+
+- CLOUDFRONT_URL.
+- CLOUDFRONT_ID.
+- CLOUDFRONT_PRIVATE_SIGNING_KEY.
+
+## Testing
+
+- In Studio add a new video component to a course.
+- Press the *Edit* button and then go to the *Advanced* section.
+- In the *Video File URLs* field add the protected video. (e,g,. /secure-cloudfront-video/?key=*secure_key*/video.mp4).
```
Comment (Squirrel18) [inline README.md]:
```suggestion
- In the *Video File URLs* field add the protected video. (e,g,. /secure-cloudfront-video/?key=*resource-id*/*resource-url*).
```

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**